scripts/custom_lint_checks.py

```diff
@@ -321,6 +345,12 @@ def check_functiondef_yields(self, node, node_doc):
                 node=node)
 
     def visit_raise(self, node):
+        """Called for raise. Adds raise message.
```

Comment:
"Visits a function node that raises an exception and verifies that all exceptions raised in the function definition are documented."

Reply:
done.